src/main/java/org/jboss/tools/intellij/analytics/PreloadLanguageServer.java

```diff
@@ -14,7 +14,7 @@
   private final ICookie cookies = ServiceManager.getService(Settings.class);
 
   private void attachLanguageClient(final File cliFile) {
-    final String[] EXTENSIONS = {"xml", "json", "txt"};
+    final String[] EXTENSIONS = {"xml", "json", "txt", "mod"};
```

Comment:
Adding Go language should not be part of this PR, you are mixing two different purposes that will render the commit log and release not difficult to read